dataland-frontend/src/components/pages/CompanyCockpitPage.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <TheHeader v-if="!useMobileView" />
   <TheContent class="paper-section flex">
-    <CompanyInfoSheet :company-id="companyId" />
+    <CompanyInfoSheet :company-id="companyId" :show-single-data-request-button="true" />
```

Comment:
Is there any scenario where the CompanyInfoSheet is called with showSingleDataRequestButton being set to "false"? Or will there be one at some point? If not, I don't see why showSingleDataRequestButton should be a property that can be passed

Reply:
Yes, if you are in a framework upload-page, then it should not be visible. Or in other words: according to the figma, the button should just be visible in the company cockpit page and in the view dataset page

Reply:
Fair